src/app/home/directives/home-series.component.ts

```diff
@@ -9,6 +9,9 @@ import { StoryModel } from '../../shared';
   styleUrls: ['home-series.component.css'],
   template: `
     <header *ngIf="noseries">
+      <section [publishHoverTip]="'These are your stories not associated with any Series.'" class="hover-tip">
```

Comment:
Could add the `hover-tip` class by using a `HostBinding('class.hover-tip') = true` in the directive, and cut down on some duplication.

If those hover tip strings get too long (enough to frighten the linter), you could start moving those to component properties.